app/boards/shields/nice_view/nice_view.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+&nice_view_spi {
+	status = "okay";
+	cs-gpios = <&pro_micro 1 GPIO_ACTIVE_HIGH>;
```

Comment:
I think this should be something like:

```suggestion
	cs-gpios = <&nice_view_d 0 GPIO_ACTIVE_HIGH>;
```

And then expect a `nice_view_d` nexus node to be supplied? Possibly with some helper macro to make it not so painful to write every time? Or with an include so you can just write:

````
&nice_view_d {
  gpio-map = <0 0 &pro_micro 1 0>;
};
```

As needed to add the mapping.